test/unit/server.spec.js

```diff
@@ -149,6 +149,21 @@ describe('server', () => {
       expect(mockConfig.port).to.be.equal(9877)
     })
 
+    it('should emit a listening event once server begin accepting connections', () => {
+      server._start(mockConfig, mockLauncher, null, mockFileList, mockWebServer, browserCollection, mockSocketServer, mockExecutor, doneSpy)
+
+      var listening = sinon.spy()
+      server.on('listening', listening)
+
+      expect(listening).not.to.have.been.called
+
+      fileListOnResolve()
+
+      server.emit('listening', 9876)
```

Comment:
This is not really testing anything is it? You should probably emit the event on the `mockWebServer` instead